openassessment/workflow/api.py

```diff
@@ -261,9 +261,11 @@ def update_from_assessments(submission_uuid, assessment_requirements):
 
     """
     workflow = _get_workflow_model(submission_uuid)
+    staff_required = assessment_requirements.get("staff", {}).get('staff', {}).get('required', False)
+    is_staff_override = workflow.staff_score_exists() and not staff_required
```

Comment:
I'm not 100% sure how to write this logic. What if we want to override a previously recorded required staff score? This current setup would not allow for that, we'd have to add an explicit `force_override` to this API method as well